app/controllers/api/todays_controller.rb

```diff
@@ -2,7 +2,10 @@ class Api::TodaysController < ApplicationController
   def index
     # workout = Workout.find_by(workout_date: Date.today)
     # render json: workout.routines
-    date = Date.today.to_s
+    # date = Date.today.to_s
+    date = Time.now.in_time_zone('Pacific Time (US & Canada)').to_date.to_s
```

Comment:
have we tested our other routes to make sure this does not create any new conflicts?